pkg/client/simple/vfsclientset/commonvfs.go

```diff
@@ -21,7 +21,7 @@ import (
 	"fmt"
 	"github.com/golang/glog"
 	kops "k8s.io/kops/pkg/apis/kops"
-	"k8s.io/kops/pkg/apis/kops/v1alpha1"
+	"k8s.io/kops/pkg/apis/kops/v1alpha2"
```

Comment:
Master was on `v1alpha1` but I needed `v1alpha2` to specify subnetIds. Am I thinking about this incorrectly?